integration/src/main/java/scripts/utils/CommonResourceFieldsUtil.java

```diff
@@ -0,0 +1,50 @@
+package scripts.utils;
+
+import bio.terra.workspace.model.AccessScope;
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import bio.terra.workspace.model.ControlledResourceCommonFields;
+import bio.terra.workspace.model.ManagedBy;
+import bio.terra.workspace.model.PrivateResourceUser;
+import bio.terra.workspace.model.Properties;
+import bio.terra.workspace.model.Property;
+import bio.terra.workspace.model.ReferenceResourceCommonFields;
+import com.google.common.collect.ImmutableMap;
+
+/** Utils to build common resource fields for both referenced resources and controlled resources. */
+public class CommonResourceFieldsUtil {
+
+  public static final ImmutableMap<String, String> DEFAULT_RESOURCE_PROPERTIES =
+      ImmutableMap.of("foo", "bar");
+
+  public static ReferenceResourceCommonFields makeReferencedResourceCommonFields(
+      String name, CloningInstructionsEnum cloningInstructions) {
+    return new ReferenceResourceCommonFields()
+        .cloningInstructions(cloningInstructions)
+        .description("Description of " + name)
+        .name(name)
+        .properties(getResourceDefaultProperties());
+  }
+
+  public static ControlledResourceCommonFields makeControlledResourceCommonFields(
```

Comment:
opt a couple callers look like

```
        CommonResourceFieldsUtil.makeControlledResourceCommonFields(
            .name(resourceName)	            resourceName,
            .cloningInstructions(CloningInstructionsEnum.NOTHING)	            /*privateUser=*/ null,
            .accessScope(AccessScope.SHARED_ACCESS)	            CloningInstructionsEnum.NOTHING,
            .managedBy(ManagedBy.USER);	            ManagedBy.USER,
            AccessScope.SHARED_ACCESS);
```

Make a variant that just takes name

Reply:
there's only one of that exact combination.